packages/crayons-core/src/components/popover/popover.tsx

```diff
@@ -170,6 +190,13 @@ export class Popover {
       this.fwHide.emit();
     }
   }
+  @Method()
+  async hide() {
+    clearTimeout(this.timerId);
```

Comment:
Can we also consider clearing timer inside disconnectedCallback as well ?

Reply:
Done